spring-cloud-alibaba-starters/spring-cloud-alibaba-commons/src/main/java/com/alibaba/cloud/commons/io/FileUtils.java

```diff
@@ -109,5 +109,26 @@ public static String readFileToString(final File file, final String encoding)
 	public static String readFileToString(final File file) throws IOException {
 		return readFileToString(file, Charset.defaultCharset());
 	}
+	
+	 public void process() {
+        SecureRandom secureRandom = new SecureRandom();
+	     System.runFinalizersOnExit(true);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Dm: Method invokes dangerous method runFinalizersOnExit (DM_RUN_FINALIZERS_ON_EXIT)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#dm-method-invokes-dangerous-method-runfinalizersonexit-dm-run-finalizers-on-exit).
*Never call System.runFinalizersOnExit or Runtime.runFinalizersOnExit for any reason: they are among the most dangerous methods in the Java libraries.* -- Joshua Bloch